velox/dwio/dwrf/reader/ReaderBase.h

```diff
@@ -62,33 +62,19 @@ class ReaderBase {
  public:
   /// Creates reader base from buffered input.
   ReaderBase(
-      memory::MemoryPool& pool,
+      const dwio::common::ReaderOptions& options,
       std::unique_ptr<dwio::common::BufferedInput> input,
-      std::shared_ptr<dwio::common::encryption::DecrypterFactory>
-          decryptorFactory = nullptr,
-      uint64_t footerEstimatedSize =
-          dwio::common::ReaderOptions::kDefaultFooterEstimatedSize,
-      uint64_t filePreloadThreshold =
-          dwio::common::ReaderOptions::kDefaultFilePreloadThreshold,
-      dwio::common::FileFormat fileFormat = dwio::common::FileFormat::DWRF,
-      bool fileColumnNamesReadAsLowerCase = false,
-      std::shared_ptr<random::RandomSkipTracker> randomSkip = nullptr,
-      std::shared_ptr<velox::common::ScanSpec> scanSpec = nullptr);
-
-  ReaderBase(
-      memory::MemoryPool& pool,
-      std::unique_ptr<dwio::common::BufferedInput> input,
-      dwio::common::FileFormat fileFormat);
+      dwio::common::FileFormat fileFormat = dwio::common::FileFormat::DWRF);
```

Comment:
This is also duplicate with `ReaderOptions::fileFormat()`

Reply:
I deleted this line, and the diff file can be seen [here](https://github.com/facebookincubator/velox/pull/10372/commits/ba8c9517656d9a7907d89806018d6f9149d3bb80)